proposals/description/tip_117.ts

```diff
@@ -0,0 +1,387 @@
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.bammDeposit, addresses.lqty, addresses.tribalCouncilTimelock, '10000'],
+      description: 'Withdraw all LQTY from B.AMM deposit to TC Timelock'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.raiPriceBoundPSM, addresses.rai, addresses.tribalCouncilSafe, '10000'],
+      description: 'Withdraw all RAI from PSM to TC Multisig where it can be sold'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [addresses.raiPriceBoundPSM, addresses.fei, addresses.daiFixedPricePSM, '10000'],
+      description: 'Withdraw all FEI from PSM to DAI PSM where it can be burnt'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.dpiToDaiLensDai, // 0$ left, auction ended
+          addresses.dpiToDaiLensDpi, // 0$ left, auction ended
+          addresses.bammDeposit, // 0$ left, deprecated (need redeploy)
+          addresses.rariPool7LusdPCVDeposit, // 0$ left
+          addresses.rariPool6FeiPCVDepositWrapper, // 114$ left
+          addresses.rariPool19FeiPCVDepositWrapper, // 0$ left
+          addresses.rariPool24FeiPCVDepositWrapper, // 6k$ left
+          addresses.agEurDepositWrapper, // no more agEUR on the timelock
+          addresses.wethDepositWrapper, // no more WETH on the timelock
+          addresses.d3poolConvexPCVDeposit, // 0$ left in Convex d3
+          addresses.d3poolCurvePCVDeposit, // 0$ left in Curve d3
+          addresses.uniswapPCVDeposit, // 0$ left in FEI/ETH Uniswap v2
+          addresses.compoundEthPCVDepositWrapper, // 0 ETH left in Compound
+          addresses.aaveEthPCVDepositWrapper, // 0 ETH left in Aave
+          addresses.raiPriceBoundPSM, // RAI will finish to be liquidated by TC multisig
+          addresses.balancerDepositFeiWeth, // 0$ left in B-70WETH-30FEI deposit
+          addresses.balancerLensBpt30Fei70Weth, // 0$ left in B-70WETH-30FEI gauge
+          addresses.ethTokemakPCVDeposit // 0$ left
+        ]
+      ],
+      description: 'Remove deprecated/empty deposits from CR oracle'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'unsetSafeAddresses(address[])',
+      arguments: (addresses) => [
+        [
+          addresses.raiPriceBoundPSM,
+          addresses.aaveRaiPCVDeposit,
+          addresses.rariPool9RaiPCVDeposit,
+          addresses.balancerDepositFeiWeth,
+          addresses.d3poolConvexPCVDeposit,
+          addresses.d3poolCurvePCVDeposit,
+          addresses.aaveEthPCVDeposit,
```

Comment:
aaveEth should stay safe imo. Its the most convenient way for PCV Guardian to wrap ETH currently

Reply:
Thinking of adding to the ERC20HoldingPCVDeposit (Erwan's idea) a method to wrap any ETH the deposit has received. In the future we could use that to wrap ETH and then withdraw it to wherever it's needed